### DISCUSSION TITLE
openmediavault login -weird error.

### DISCUSSION THREAD

**Discussion by piraterx** [linuxserver/Heimdall#750] [App issues]
I have heimdall on one of raspberry docker container. I am using my openmediavault which on another server 192.168.1.191. I am putting http://192.168.1.191/ on heimdall. If I click on it, takes me to login page. I put my credential and click log in. It brings back me to same page. I put again. same thing. Now if I type http://192.168.1.191/ and put credential, it takes me to OMV UI main page where it should. 

Why? what difference it is making me clicking on heimdall link that it doesn't let me login? Another interesting thing is, if I have OMV logged in before I click on heimdall link, it even log me out other instances which were all ready open.

**Comment by k-matti** [comment]
I have the same problem. Did you resolve it?

**Comment by johnrowland** [comment]
Same issue with my setup.
Hardware is x86
Container on macvlan, openmediavault (5.5) is docker host.
Host <--> Container communication verified (successful pings to IPs).

Edit 1:
Error occurs when accessed using Firefox browser (84.0) on a 32-bit Lubuntu system.
Works fine (can login and get to OMV webUI) when accessed using Chrome on Android phone.

**Comment by piraterx** [comment]
> 
> 
> I have the same problem. Did you resolve it?

I believe it has to do with browser. sorry for later reply. Didn't login till now. I used chrome browser and works fine.

**Comment by jsholovitz** [comment]
Can confirm the issue here. Agree that it seems browser-specific. I can confirm that it occurs in Firefox 87.0 but not in the latest Edge.
